app/components/works/add_files_component.html.erb

```diff
@@ -122,7 +122,7 @@
                 <p>This will move your files from Globus to SDR.</p>
               </div>
             </li>
-          </ul>
+          </ol>
```

Comment:
I assume this is a11y-related and it matches a preceding `<ol>`?

Reply:
Yes.